### DISCUSSION TITLE
Rancher desktop on windows Kubernetes error

### DISCUSSION THREAD

**Discussion by EdeBont** [rancher-sandbox/rancher-desktop#1930] [General]
After a successful installation I get an error on starting Rancher desktop.
Error Starting Kubernetes -> Certificate validation error: UNABLE_TO_GET_ISSUER_CERT_LOCALLY

Have been googling for 2 days and cannot find an answer.

Anybody?

**Comment by pln-djn** [comment]
Hey, try version [1.1.0](https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.1.0) that solved it for me.

**Reply by EdeBont** [reply]
Already tried that because that was the version that initially worked for me.
But after uninstalling Rancher Desktop and installing version 1.1.0 I got the same problem.

**Comment by jcageman** [comment]
Also struggling with this issue (also upgraded from 1.1.0). There is already an open issue for it, but the workaround there doesn't solve my problem: https://github.com/rancher-sandbox/rancher-desktop/issues/1918

**Reply by EdeBont** [reply]
Glad I'm not the only one.
I also looked into #1918 but for it does not work for me.

**Comment by EdeBont** [comment]
Downgrading reinstalling version 1.0.0 finally worked for me.
As soon as I upgrade stuff breaks.
